### PR TITLE
feat(judicial-system): Upload to Auður

### PR DESCRIPTION
**Pull request by oddsson**
# Upload to Auður

https://app.asana.com/0/1199153462262248/1200905978520084/f

## What

Allow judges and registrars to upload case files to Auður.

## Why

Today, they are opening the files, saving them on their computers and manually uploading them to Auður. This feature significantly improves security and user experience.

## Screenshots / Gifs

https://user-images.githubusercontent.com/3789875/134362708-ddc5a91d-86d9-40cb-821f-9cfda29f7962.mov

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Comment by oddsson** [comment]
In the video in the description, the first file name is really long and it doesn't look good. I've created a task for this in Asana because it's not a blocker for this PR.

File: apps/judicial-system/web/src/shared-components/CaseFileList/CaseFileList.tsx [hunk 1]
```diff
@@ -1,37 +1,68 @@
...
+            <UploadedFile
+              file={file}
+              showFileSize={true}
+              defaultBackgroundColor="blue100"
+              doneIcon="checkmark"
+              hideIcons={
+                hideIcons ||
+                [
+                  CaseFileState.BOKEN_LINK,
+                  CaseFileState.DELETED,
```
Comment (gudjong) [inline apps/judicial-system/web/src/shared-components/CaseFileList/CaseFileList.tsx:45]: The server will never return DELETED files.

Comment (gudjong) [reply apps/judicial-system/web/src/shared-components/CaseFileList/CaseFileList.tsx:45]: Consider removing DELETED test here.

**Review by gudjong — CHANGES_REQUESTED** [review]
One minor change request consideration.

**Review by gudjong — APPROVED** [review]

**Pull request merged**